scripts/docker_deploy.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+version="0.4.10"
```

Comment:
there are parts that are hardcoded but I understand your point. Maybe this upcoming commit will work?